sympy/ntheory/modular.py

```diff
@@ -250,3 +250,52 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+def mod_inverse(a, m):
+    """
+    Given two number a  and m.This function computes the number
+    c such that   (a*c)%m==1.Meaning that if a is multiplied by c
+    then the remainder obtained on division with m is 1.
+    The value of m need not be a prime.This function  returns
+    the value of c  if some c exists.Otherwies it raises an
+    Exception stating that modular inverse does not exist.
+
+    Examples
+    ========
+    >>> from sympy.ntheory.modular import mod_inverse
+    >>> mod_inverse(3,11)
+    4
+
+    Suppose we wish to find modular multiplicative inverse x of
+    3 modulo 11.This is the same as finding x such that 3*x = 1 (mod 11)
+    we find one value of x that satisfies this congruence is 4.
+    Because 3*4=12   and 12 = 1 mod(11)
```

Comment:
I would rewrite this whole paragraph to:

Suppose we want to find the multiplicative inverse of 3 modulo 11. In other words, we want to find x such that `3 \times x \equiv 1 mod 11`. Since `3 \times 4 = 12 \equiv 1 mod 11` we know that `x = 4`.